pom.xml

```diff
@@ -29,20 +29,25 @@
 
     <properties>
         <argLine>-Djava.security.egd=file:/dev/./urandom</argLine>
-        <commons-io.version>2.6</commons-io.version>
+        <commons-io.version>2.7</commons-io.version>
         <docker-maven-plugin.version>1.0.0</docker-maven-plugin.version>
         <docker.directory>docker-ready</docker.directory>
         <docker.registry>ibmcom</docker.registry>
+        <guava.version>30.0-jre</guava.version>
         <helm.chart.name>${project.artifactId}</helm.chart.name>
         <helm.download.url>https://get.helm.sh/${helm-client-executable}</helm.download.url>
         <helm.repo.addr>https://repo.helm.com</helm.repo.addr>
         <helm.repo.api.url>${helm.repo.addr}/api/accanto</helm.repo.api.url>
         <helm.repo.snapshots.url>${helm.repo.addr}/accanto/snapshots</helm.repo.snapshots.url>
         <helm.repo.url>${helm.repo.addr}/accanto/stable</helm.repo.url>
+        <httpclient.version>4.5.13</httpclient.version>
+        <jackson.version>2.13.1</jackson.version>
         <jacoco-maven-plugin.version>0.8.4</jacoco-maven-plugin.version>
         <java.version>1.8</java.version>
-        <kafka.version>2.2.1</kafka.version>
+        <kafka.version>2.4.1</kafka.version>
         <kiwigrid.version>2.5</kiwigrid.version>
+        <log4j-api.version>2.13.2</log4j-api.version>
```

Comment:
was there a reason log4j-api hasn't also been raised to 2.13.2? Would it make sense to match it up to the core version?

Reply:
Hi @clloydaccanto 

Updated Log4j-api & Log4j-core versions  to 2.17.1 .

The Build is Successful and twist locks scan report is also doesn't show any additional  vulnerabilities .

Reply:
Hi @clloydaccanto 

Latest Build Details : 

![image](https://user-images.githubusercontent.com/96560894/149067994-80f3a9a4-7e9b-4190-8b6a-0bdda3dba0f2.png)

Cluster Installation Details 
![image (2)](https://user-images.githubusercontent.com/96560894/149068142-d22b42ce-109d-415f-b2ee-d312166abb47.png)